corehq/apps/reports/filters/forms.py

```diff
@@ -622,14 +617,14 @@ class CompletionOrSubmissionTimeFilter(BaseSingleOptionFilter):
     default_text = gettext_lazy("Completion Time")
 
     def _generate_help_message():
-        completion_help = mark_safe_lazy(gettext_lazy(  # nosec: no user input
-            "<strong>Completion</strong> time is when the form is completed on the phone."))
+        completion_help = gettext_lazy(  # nosec: no user input
+            "<strong>Completion</strong> time is when the form is completed on the phone.")
 
-        submission_help = mark_safe_lazy(gettext_lazy(  # nosec: no user input
+        submission_help = gettext_lazy(  # nosec: no user input
             "<strong>Submission</strong> time is when {hq_name} receives the form.".format(
-                hq_name=commcare_hq_names()['commcare_hq_names']['COMMCARE_HQ_NAME'])))
+                hq_name=commcare_hq_names()['commcare_hq_names']['COMMCARE_HQ_NAME']))
```

Comment:
Not marking these safe doesn't result in the html getting (incorrectly) escaped when it's presented?

Reply:
Ah yeah I didn't fully understand the change I was making. You are correct. I should have realized the order of operations here was important! I'll change it back to using `mark_safe_lazy`.

Reply:
See 0f72f3dc3ae1358617ffc5436a6e3f38e53c130e